### DISCUSSION TITLE
My subscription has ended, and I can't renew it.

### DISCUSSION THREAD

**Discussion by Osamamicro** [community/community#49072] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I did a request to buy the yearly plan subscription and add my credit card info I  get a congratulation screen but in the vs code still did not activate and my balance didn't change

**Comment by iamcjwilliams** [comment]
@Osamamicro Have you viewed this doc to troubleshoot your issue? https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Reply by AnikoV** [reply]
The same. Nothing help in troubleshoot

**Reply by AnikoV** [reply]
Still not enabled in accout settings. money was not debited from the account.
